File: Baekjoon_Online_Judge/NO_01110/1110.cpp
#include <iostream>
using namespace std;
/*
 -------------------------------------- 
 | Created by happiness96             | 
 | Year 2020                          | 
 | Month 09                           | 
 | Day 01                             | 
 | 더하기 사이클                      | 
 -------------------------------------- 
*/

int main(){
    int N, ori, cycle_length = 0;

    cin >> N;
    ori = N;

    while (true){
        cycle_length ++;

        N = N % 10 * 10 + (N % 10 + N / 10) % 10;

        if (N == ori) break;
    }

    cout << cycle_length;

    return 0;
}

File: Baekjoon_Online_Judge/NO_01110/1110.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# -------------------------------------- #
# | Created by happiness96             | #
# | Year 2020                          | #
# | Month 09                           | #
# | Day 01                             | #
# | 더하기 사이클                      | #
# -------------------------------------- #


if __name__ == "__main__":
    N = int(r_input())
    ori = N

    cycle_length = 0

    while True:
        cycle_length += 1

        N = N % 10 * 10 + (N % 10 + N // 10) % 10

        if N == ori:
            break
    
    print(cycle_length)
